### PR TITLE
Replace Home Profile Activity Summary by Type chart with Plotly.js #317

### LINKED ISSUE #317: Replace Home Profile Activity Summary by Type chart with Plotly.js
On the Home Profile page, there is a chart called 'Activity Summary by Type':
screenshot from 2019-01-27 11-22-16

The chart is made with NVD3, which we are in the process of deprecating.

# Task
Replace the NVD3 version of the Home Profile Activity Summary by Type chart with Plotly.js. Make sure the functionality is mainly the same. Also, check for legend overlap on small screens, and fix if it happens.

- [ ] NVD3 chart is replaced with Plotly.js
- [ ] legend does not overlap chart on small screens
- [ ] interacting with legend allows chart to be filtered

BONUS: chart can be toggled to show *activity counts* or *activity minutes* (the data are already available).

### PR DESCRIPTION
**Pull request by marla-singer**
Closes #317 

### Compare different screen sizes (the old chart presents)

screenshot at jan 29 16-45-48
screenshot at jan 29 16-43-13
screenshot at jan 29 16-44-48

### CODE REVIEW

File: app/client/views/home/activities/sums/sums.js [hunk 1]
```diff
@@ -1,59 +1,86 @@
...
-    .stacked(true)
-    .showControls(false)
-    .color(customColors)
-
-  activityCountChart
-    .yAxis.axisLabel("Activity count")
-    .tickFormat(d3.format('d'));
-
-  instance.autorun(function () {
+  templateInstance.autorun(function () {
```
Comment (brylie) [inline app/client/views/home/activities/sums/sums.js]: Make sure the chart rendering autorun is within the `Template.onRendered` callback, for safety.

Comment (marla-singer) [reply app/client/views/home/activities/sums/sums.js]: What do you mean? 

1) It's the general part in whole project where chart is rendered
2) Before rendering it makes sure data exists

Please help me What do I need to check additionally?

Comment (brylie) [reply app/client/views/home/activities/sums/sums.js]: IIs this in an onCreated or onRendered autorun definition?

Comment (brylie) [reply app/client/views/home/activities/sums/sums.js]: If the chart-render autorun is defined in an onCreated callback, just move that autorun to an onRendered callback. That will ensure DOM is ready, even though the data method will be slow. Just setting up a good habit: anything that relies on DOM should be guaranteed to run after DOM is ready.

Comment (brylie) [reply app/client/views/home/activities/sums/sums.js]: Sorry, I was reading this on mobile and didn't see the whole file.

**Pull request merged**